src/utils/handleAndSignImages.ts

```diff
@@ -0,0 +1,32 @@
+import isSVGImage from '@/utils/isSVG';
+import { getFullSizeUrl } from '@/utils/getFullSizeUrl';
+import { maybeSignUri } from '@/handlers/imgix';
+import svgToPngIfNeeded from '@/handlers/svgs';
+import { getLowResUrl } from '@/utils/getLowResUrl';
+
+export function handleAndSignImages(
+  imageUrl?: string,
+  previewUrl?: string,
+  originalUrl?: string
+): {
+  imageUrl?: string;
+  lowResUrl?: string;
+} {
+  const image = imageUrl || originalUrl || previewUrl;
+
+  if (!image) {
+    return {};
+  }
+
+  const isSVG = isSVGImage(image);
+  const fullImage = isSVG ? image : getFullSizeUrl(image);
+
+  const lowResUrl = isSVG
+    ? maybeSignUri(svgToPngIfNeeded(image, false), { w: 150 })
```

Comment:
reason for not using CardSize here? i get spooked having a bunch of diff sized images cached in memory

Reply:
not sure how CardSize went missing here tbh, i will put it back

Reply:
Oh I removed it. CardSize is dependent on device width, so it will end up with a bunch of sizes cached in memory. With this, I tried to hit a compromise and have a single width cached in memory.

Thoughts?

Reply:
Perhaps `handleAndSignImages` should have some presets. The 150 is based on what a low-res CardSize would be on average phones.

Reply:
wouldn't only 1 card size be cached in memory per device, since each device only has 1 set width?

Reply:
Oh sorry, not in memory, in the CDN. We pay for each transform. Maybe doesn't matter, but if we're blurring a low-res image anyway a handful of preset image widths is pry fine either way.

This fn was used in a couple places, so assumed it was general purpose. If that's the case, then maybe we should also update it to accept a `width` prop, and pass in `CardSize` in this particular case.

Don't derail for this though, we can put CardSize back for now.